src/main/java/org/wso2/carbon/esb/connector/utils/SalesforceConstants.java

```diff
@@ -51,6 +51,21 @@ public class SalesforceConstants {
     public static final String CLOSE_JOB_URL = "closeJobUrl";
     public static final String ABORT_JOB_URL = "abortJobUrl";
     public static final String ABORT_QUERY_JOB_URL = "abortQueryJobUrl";
+    public static final String CREATE_JOB_URL = "createJobUrl";
+    public static final String CREATE_QUERY_JOB_URL = "createQueryJobUrl";
+    public static final String DELETE_JOB_URL = "deleteJobUrl";
+    public static final String DELETE_QUERY_JOB_URL = "deleteQueryJobUrl";
+    public static final String ALL_JOB_INFO_URL = "allJobInfoUrl";
+    public static final String ALL_QUERY_JOB_INFO_URL = "allQueryJobInfoUrl";
+    public static final String GET_FAILED_RESULTS_URL = "getFailedResultsUrl";
+    public static final String JOB_INFO_URL = "jobInfoUrl";
+    public static final String QUERY_JOB_INFO_URL = "queryJobInfoUrl";
+    public static final String GET_QUERY_JOB_RESULTS_URL = "getQueryJobResultsUrl";
+    public static final String GET_SUCCESSFUL_RESULTS_URL = "getSuccessfulResultsUrl";
+    public static final String GET_UNPROCESSED_RESULTS_URL = "getUnprocessedResultsUrl";
+    public static final String UPLOAD_JOB_DATA_URL = "uploadJobUrl";
+    public static final String TOKEN_URL = "tokenUrl";
+    public static final String PAYLOAD = "payload";
```

Comment:
payload is too simple, it can easily get conflict. Shall we put some name with a prefix. like sf__bulk_payload, let's add the prefix to all such properties we are setting to the messageContext. 
ex: SF_OAUTH_CONFIG_NAME = "name"; should be changed to SF_OAUTH_CONFIG_NAME = "sf_bulk_name";